src/main/webapp/tsconfig.json

```diff
@@ -18,6 +18,7 @@
   },
   "angularCompilerOptions": {
     "fullTemplateTypeCheck": true,
-    "strictInjectionParameters": true
+    "strictInjectionParameters": true,
+    "enableIvy": false
```

Comment:
Is there a particular reason you need to add this?

Reply:
Nope I added that while trying to debug some of my errors and forgot to take it out.